cli/ioctl/cmd/config/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var configFileName string
+
+// ConfigCmd represents the config command
+var ConfigCmd = &cobra.Command{
+	Use:       "config",
+	Short:     "Set or get configuration for ioctl",
+	ValidArgs: []string{"set", "get"},
+	Args:      cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Print: " + strings.Join(args, " "))
+	},
+}
+
+func init() {
+	dir := os.Getenv("HOME") + "/.config/ioctl"
+	if err := os.MkdirAll(dir, 0700); err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	configFileName = dir + "/config.default"
```

Comment:
Create the dir here is not great as user won't run "ioctl config" first to use "ioctl config set endpoint 1.2.3.4". The dir creation logic should exist in "ioctl config set" cmd.

Reply:
user don't need to run "ioctl config" first, tested that directly run "ioctl config set endpoint 1.2.3.4" can work, golang will run all init() under /config dir
so i think it could be here. if we move to set cmd we should also move to get cmd, which is not necessary

Reply:
also we'll have other configs, do it here then we don't need to worry later when adding other cmds under /config